man/mono.1

```diff
@@ -1652,11 +1652,11 @@ Automatically generates sequence points where the
 IL stack is empty.  These are places where the debugger can set a
 breakpoint.
 .TP
-\fBgen-compact-seq-points\fR
```

Comment:
thanks for the input. In the new commits both `MONO_DEBUG=gen-compact-seq-points` and `--aot=gen-seq-points-file=<dir>` are still available but with deprecated warning.